client/stripe-utils/utils.js

```diff
@@ -23,8 +23,8 @@ import {
 const getStripeServerData = () => {
 	// Classic checkout.
 	// eslint-disable-next-line camelcase
-	if ( ! wc_stripe_upe_params ) {
-		throw new Error( 'Stripe initialization data is not available' );
+	if ( typeof wc_stripe_upe_params === 'undefined' ) {
+		return {};
```

Comment:
Question to the reviewer: Can you think of any case where returning an empty object may create problems?

Reply:
That's a good question. I reviewed all the `getStripeServerData`, and none of them are wrapped in a try...catch with the purpose of catching the previous exception. So, the only use for the exception would be to prevent the execution of the rest of the code.

Removing this "pause" could cause some weird behaviors, such as not having the account country, currency, cart totals, etc. This could potentially allow customers to make free purchases and use payment methods that should not be available, among other things (at least in theory).

I would avoid this change unless it is really required for the main fix.